src/app/quizPage.js

```diff
@@ -0,0 +1,222 @@
+import {
+    QuestionService
+} from "../service/QuestionService.js"
+
+import {
+    QUIZ_PAGE_STYLES,
+    START_PAGE_STYLES,
+    TIMEOUT_AFTER_ANSWER_SELECTION
+} from "./appSettings.js"
+
+// will be filledi with mode object during page rendering
+let CURRENT_MODE = null; 
+let GENERATOR = null;
+
+// use to render the page for the first time, after the game start
+export function renderQuizPage(mode) {
+    const appScreen = document.querySelector('#pokequiz-app');
+    appScreen.classList.add(QUIZ_PAGE_STYLES.quizPageClass)
+    appScreen.classList.remove(START_PAGE_STYLES.startPageClass)
+    const quizTemplate = document.getElementById('quiz-template');
+    appScreen.innerHTML = quizTemplate.innerHTML;
+    // TODO later  - generate question using questionService - below are temporary dummy variables
+    const generatedQuestion = {
+        question: "quizQuestion",
+        questionNum: 1,
+    }
+    setupPageTitle(mode);
+    CURRENT_MODE = mode;
```

Comment:
maybe set it at the top of the function and then only refer to `CURRENT_MODE`?

Reply:
right, I changed it in the last line, and forgot here